### ISSUE TITLE
BLE-MESH: esp_bt_controller_init failed when using arduino-esp32 as a component in project (IDFGH-9607)

### ISSUE THREAD

**Issue by hipal31** [espressif/esp-idf#10955]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### IDF version.

v4.4.4

### Operating System used.

Windows

### How did you build your project?

Command line with idf.py

### Development Kit.

ESP32C3, ESP32

### Power Supply used.

External 3.3V

### What is the expected behavior?

When I add arduino-esp32 as a component of a ble-mesh project then there must no error on runtime.

### What is the actual behavior?

In my project, I'm developing product using BLE-Mesh. And I need arduino-esp32 as a component in my project. When I add arduino-esp32 as a component then `esp_bt_controller_init ` failed to initialize.

### Steps to reproduce.

1. Take a onoff_server example of ble-mesh
2. Add arduino-esp32(v2.0.7) as a component
3. Compile it 
4. Flash it on esp32c3 board (also try on esp32)

### Debug Logs.

```plain
ESP-ROM:esp32c3-api1-20210207
Build:Feb  7 2021
rst:0x1 (POWERON),boot:0xc (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fcd5810,len:0x16cc
load:0x403cc710,len:0x944
load:0x403ce710,len:0x2dfc
entry 0x403cc710
I (30) boot: ESP-IDF v4.4.4-1-g8a624b9b5f-dirty 2nd stage bootloader
I (30) boot: compile time 14:20:18
I (30) boot: chip revision: v0.3
I (34) boot.esp32c3: SPI Speed      : 80MHz
I (39) boot.esp32c3: SPI Mode       : DIO
I (43) boot.esp32c3: SPI Flash Size : 4MB
I (48) boot: Enabling RNG early entropy source...
I (54) boot: Partition Table:
I (57) boot: ## Label            Usage          Type ST Offset   Length
I (64) boot:  0 sec_cert         unknown          3f 06 0000d000 00003000
I (72) boot:  1 nvs              WiFi data        01 02 00010000 00006000
I (79) boot:  2 otadata          OTA data         01 00 00016000 00002000
I (87) boot:  3 phy_init         RF data          01 01 00018000 00001000
I (94) boot:  4 ota_0            OTA app          00 10 00020000 001db000
I (102) boot:  5 ota_1            OTA app          00 11 00200000 001db000
I (109) boot:  6 fctry            WiFi data        01 02 003db000 00006000
I (117) boot: End of partition table
I (121) boot: No factory image, trying OTA 0
I (126) esp_image: segment 0: paddr=00020020 vaddr=3c170020 size=44d98h (282008) map
I (179) esp_image: segment 1: paddr=00064dc0 vaddr=3fc94c00 size=04f20h ( 20256) load
I (183) esp_image: segment 2: paddr=00069ce8 vaddr=40380000 size=06330h ( 25392) load
I (190) esp_image: segment 3: paddr=00070020 vaddr=42000020 size=16edfch (1502716) map
I (433) esp_image: segment 4: paddr=001dee24 vaddr=40386330 size=0e744h ( 59204) load
I (451) boot: Loaded app from partition at offset 0x20000
I (493) boot: Set actual ota_seq=1 in otadata[0]
I (494) boot: Disabling RNG early entropy source...
I (505) cpu_start: Pro cpu up.
I (513) cpu_start: Pro cpu start user code
I (513) cpu_start: cpu freq: 160000000
I (513) cpu_start: Application information:
I (516) cpu_start: Project name:     BLE-IR-BLASTER
I (522) cpu_start: App version:      8386fda-dirty
I (527) cpu_start: Compile time:     Mar  9 2023 16:20:08
I (533) cpu_start: ELF file SHA256:  3895d7f8f56ffd09...
I (539) cpu_start: ESP-IDF:          v4.4.4-1-g8a624b9b5f-dirty
I (546) heap_init: Initializing. RAM available for dynamic allocation:
I (553) heap_init: At 3FCA6B50 len 00035BC0 (214 KiB): DRAM
I (559) heap_init: At 3FCDC710 len 00002950 (10 KiB): STACK/DRAM
I (566) heap_init: At 50000020 len 00001FE0 (7 KiB): RTCRAM
I (573) spi_flash: detected chip: generic
I (577) spi_flash: flash io: dio
I (581) sleep: Configure to isolate all GPIO pins in sleep state
I (587) sleep: Enable automatic switching of GPIO sleep configuration
I (595) coexist: coexist rom version 9387209
I (600) cpu_start: Starting scheduler.
I (613) gpio: GPIO[5]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:1
I (619) DEVICE: REASON : 1
I (621) DEVICE: COUNT : 0
I (625) REEVA_BLE: Initializing...
I (629) gpio: GPIO[18]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0
btdm: bss start 0x3fcdf0a8, len 40
btdm: data start 0x3fcdf09c, data start rom 0x400591fc, len 12
MAGIC fadebead VERSION 00010006
I (649) BTDM_INIT: BT controller compile version [02bfb7f]
I (656) phy_init: phy_version 950,11a46e9,Oct 21 2022,08:56:12
W (662) phy_init: failed to load RF calibration data (0x1102), falling back to full calibration
E (714) EXAMPLE_INIT: bluetooth_init initialize controller failed
E (715) REEVA_BLE: esp32_bluetooth_init failed (err 257 = ESP_ERR_NO_MEM)
I (718) REEVA_IR: Restore, key "AC_DATA" not exists
I (723) REEVA_IR: IR STARTED ...
I (727) gpio: GPIO[19]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0
E (737) gpio: gpio_install_isr_service(449): GPIO isr service already installed
I (871) REEVA_IR: DECODER UNKNOWN 0x77DB229F 37
I (10626) DEVICE: Erasing Count .....
ESP-ROM:esp32c3-api1-20210207
Build:Feb  7 2021
rst:0x1 (POWERON),boot:0xc (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fcd5810,len:0x16cc
load:0x403cc710,len:0x944
load:0x403ce710,len:0x2dfc
entry 0x403cc710
I (30) boot: ESP-IDF v4.4.4-1-g8a624b9b5f-dirty 2nd stage bootloader
I (30) boot: compile time 14:20:18
I (30) boot: chip revision: v0.3
I (34) boot.esp32c3: SPI Speed      : 80MHz
I (39) boot.esp32c3: SPI Mode       : DIO
I (43) boot.esp32c3: SPI Flash Size : 4MB
I (48) boot: Enabling RNG early entropy source...
I (54) boot: Partition Table:
I (57) boot: ## Label            Usage          Type ST Offset   Length
I (64) boot:  0 sec_cert         unknown          3f 06 0000d000 00003000
I (72) boot:  1 nvs              WiFi data        01 02 00010000 00006000
I (79) boot:  2 otadata          OTA data         01 00 00016000 00002000
I (87) boot:  3 phy_init         RF data          01 01 00018000 00001000
I (94) boot:  4 ota_0            OTA app          00 10 00020000 001db000
I (102) boot:  5 ota_1            OTA app          00 11 00200000 001db000
I (109) boot:  6 fctry            WiFi data        01 02 003db000 00006000
I (117) boot: End of partition table
I (121) esp_image: segment 0: paddr=00020020 vaddr=3c170020 size=44d98h (282008) map
I (175) esp_image: segment 1: paddr=00064dc0 vaddr=3fc94c00 size=04f20h ( 20256) load
I (178) esp_image: segment 2: paddr=00069ce8 vaddr=40380000 size=06330h ( 25392) load
I (185) esp_image: segment 3: paddr=00070020 vaddr=42000020 size=16edfch (1502716) map
I (428) esp_image: segment 4: paddr=001dee24 vaddr=40386330 size=0e744h ( 59204) load
I (446) boot: Loaded app from partition at offset 0x20000
I (447) boot: Disabling RNG early entropy source...
I (458) cpu_start: Pro cpu up.
I (467) cpu_start: Pro cpu start user code
I (467) cpu_start: cpu freq: 160000000
I (467) cpu_start: Application information:
I (470) cpu_start: Project name:     BLE-IR-BLASTER
I (475) cpu_start: App version:      8386fda-dirty
I (481) cpu_start: Compile time:     Mar  9 2023 16:20:08
I (487) cpu_start: ELF file SHA256:  3895d7f8f56ffd09...
I (493) cpu_start: ESP-IDF:          v4.4.4-1-g8a624b9b5f-dirty
I (500) heap_init: Initializing. RAM available for dynamic allocation:
I (507) heap_init: At 3FCA6B50 len 00035BC0 (214 KiB): DRAM
I (513) heap_init: At 3FCDC710 len 00002950 (10 KiB): STACK/DRAM
I (520) heap_init: At 50000020 len 00001FE0 (7 KiB): RTCRAM
I (526) spi_flash: detected chip: generic
I (531) spi_flash: flash io: dio
I (535) sleep: Configure to isolate all GPIO pins in sleep state
I (541) sleep: Enable automatic switching of GPIO sleep configuration
I (548) coexist: coexist rom version 9387209
I (554) cpu_start: Starting scheduler.
I (568) gpio: GPIO[5]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:1
I (572) DEVICE: REASON : 1
I (575) DEVICE: COUNT : 0
I (579) REEVA_BLE: Initializing...
I (583) gpio: GPIO[18]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0
btdm: bss start 0x3fcdf0a8, len 40
btdm: data start 0x3fcdf09c, data start rom 0x400591fc, len 12
MAGIC fadebead VERSION 00010006
I (603) BTDM_INIT: BT controller compile version [02bfb7f]
I (610) phy_init: phy_version 950,11a46e9,Oct 21 2022,08:56:12
MAGIC fadebead VERSION 00010001
Guru Meditation Error: Core  0 panic'ed (Instruction access fault). Exception was unhandled.

Stack dump detected
Core  0 register dump:
MEPC    : 0x00000100  RA      : 0x40021926  SP      : 0x3fca9820  GP      : 0x3fc95400
TP      : 0x3fc65658  T0      : 0x4005890e  T1      : 0x0000000f  T2      : 0x40024022
S0/FP   : 0x3fcdff9c  S1      : 0x3fce0000  A0      : 0x00000028  A1      : 0x3fcdff60
A2      : 0x0000006c  A3      : 0x00000001  A4      : 0x3fc9d000  A5      : 0x00000101
A6      : 0x80000000  A7      : 0x00000010  S2      : 0x3fcacbe4  S3      : 0x3fce0000
S4      : 0x3fcdf96c  S5      : 0x3fce0000  S6      : 0x00000000  S7      : 0x00000000
S8      : 0x00000000  S9      : 0x00000000  S10     : 0x00000000  S11     : 0x00000000
T3      : 0x00000000  T4      : 0x40030df4  T5      : 0x40030bda  T6      : 0x400286d4
MSTATUS : 0x00001881  MTVEC   : 0x40380001  MCAUSE  : 0x00000001  MTVAL   : 0x00000100
0x40380001: _vector_table at ??:?

MHARTID : 0x00000000

ClearCommError failed (PermissionError(13, 'Access is denied.', None, 5))
Waiting for the device to reconnect......Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'd:\\development\\ble-mesh\\ble-ir-blaster\\build\\BLE-IR-BLASTER.elf', '-ex', 'target remote | "C:\\Espressif\\python_env\\idf4.4_py3.8_env\\Scripts\\python.exe" "D:\\Frameworks\\esp-idf-v4.4.2\\tools\\idf_monitor_base\\..\\gdb_panic_server.py" --target esp32c3 "C:\\Users\\HIPAL_~1\\AppData\\Local\\Temp\\tmpa_c8cp4o"', '-ex', 'bt']' returned non-zero exit status 1.
b"error starting child process 'C:\\Espressif\\python_env\\idf4.4_py3.8_env\\Scripts\\python.exe D:\\Frameworks\\esp-idf-v4.4.2\\tools\\idf_monitor_base\\..\\gdb_panic_server.py --target esp32c3 C:\\Users\\HIPAL_~1\\AppData\\Local\\Temp\\tmpa_c8cp4o': CreateProcess: No such file or directory\r\nNo stack.\r\n"


Core  0 register dump:
MEPC    : 0x00000100  RA      : 0x40021926  SP      : 0x3fca9820  GP      : 0x3fc95400
TP      : 0x3fc65658  T0      : 0x4005890e  T1      : 0x0000000f  T2      : 0x40024022
S0/FP   : 0x3fcdff9c  S1      : 0x3fce0000  A0      : 0x00000028  A1      : 0x3fcdff60
A2      : 0x0000006c  A3      : 0x00000001  A4      : 0x3fc9d000  A5      : 0x00000101
A6      : 0x80000000  A7      : 0x00000010  S2      : 0x3fcacbe4  S3      : 0x3fce0000
S4      : 0x3fcdf96c  S5      : 0x3fce0000  S6      : 0x00000000  S7      : 0x00000000
S8      : 0x00000000  S9      : 0x00000000  S10     : 0x00000000  S11     : 0x00000000
T3      : 0x00000000  T4      : 0x40030df4  T5      : 0x40030bda  T6      : 0x400286d4
MSTATUS : 0x00001881  MTVEC   : 0x40380001  MCAUSE  : 0x00000001  MTVAL   : 0x00000100
MHARTID : 0x00000000

Stack memory:
3fca9820: 0x3fce0000 0x3fce0000 0x3fce0000 0x420c22c2 0x3fce0000 0x3fcdf96c 0x3fce0000 0x420bfd96
3fca9840: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000001 0x3fc9da50 0x4038212a
3fca9860: 0x3fcdf980 0x00000004 0x5072b434 0x4038000a 0x00000000 0x00000001 0x00000000 0x00000000
3fca9880: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000001 0x3fca98d8 0x4203dd78
3fca98a0: 0x00000000 0x00000000 0x00000000 0x3fca98cc 0x3c1765e0 0x3c176530 0x00000003 0x420cd2c6
3fca98c0: 0x00000000 0x00000000 0x00000001 0x4200fed4 0x3c19dba4 0x3fca9928 0x5a5aa5a5 0x02209230
3fca98e0: 0x00171000 0x00000a01 0x001e0000 0x01000000 0x00000000 0x000e0000 0x00000001 0x00640001
3fca9900: 0x00000064 0x01010000 0x00140005 0x00000000 0x00000000 0x00000000 0x00000001 0x4200b860
3fca9920: 0x00000000 0x00000000 0x00040000 0x00000000 0x00000002 0x00000000 0x00000000 0x00000000
3fca9940: 0x00000000 0x00000000 0x00000001 0x42009414 0x00000000 0x00000000 0x00000000 0x00000000
3fca9960: 0x00000000 0x00000006 0x00000000 0x3fca9a94 0x3fcab360 0x3fcab680 0x3fcab680 0x420d5000
3fca9980: 0x00001800 0x00000200 0x00000000 0x00000045 0xff010100 0xb3e3e6a5 0x445f4341 0x00415441
3fca99a0: 0x00000000 0x00000000 0xffffff04 0xffffffff 0x00024203 0xb9f0f350 0x5f6c6163 0x0063616d
3fca99c0: 0x00000000 0x00000000 0xffff0006 0xcdf8388f 0x00024203 0xb9f0f350 0x5f6c6163 0x0063616d
3fca99e0: 0x00000000 0x00000000 0xffff0006 0xcdf8388f 0x00024203 0xb9f0f350 0x5f6c6163 0x0063616d
3fca9a00: 0x00000000 0x00000000 0x00000002 0xcdf8388f 0x00024203 0xb9f0f350 0x5f6c6163 0x0063616d
3fca9a20: 0x00000000 0x00000000 0x00000000 0x3fcab2f0 0x00000000 0x3fcab680 0x3fca6b50 0x403916f2
3fca9a40: 0x00000000 0x3fcab2f0 0x3fcab680 0x40380eac 0x00000000 0x00000000 0x00000000 0x4039260c
3fca9a60: 0x00000000 0x3fcab2f0 0x00000000 0x00000024 0x00000000 0x00000024 0x3fca6b50 0x4039169a
3fca9a80: 0x3c171d7c 0x3fcdf980 0x3fcdf96c 0x3fcab660 0x00000650 0x3fcdf980 0x3fcdf980 0x40390512
3fca9aa0: 0x3c171d7c 0x00000650 0x3fcdf96c 0x40391604 0x3c171d7c 0x3fcdffd0 0x3fcdf96c 0x42005014
3fca9ac0: 0x00000000 0x3fca77ec 0x00000000 0x00000000 0x00000000 0x00000000 0x00000020 0x00000000
3fca9ae0: 0x00000001 0x00000001 0x00000000 0x00000001 0x00000000 0x0000000c 0x00000000 0x00000000
3fca9b00: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x42007df2
3fca9b20: 0x00000000 0x00000000 0x00000000 0x4216e9f2 0x00000000 0x00000000 0x00000000 0x4038e614
3fca9b40: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0xa5a5a5a5 0xa5a5a5a5 0xa5a5a5a5
3fca9b60: 0xa5a5a5a5 0x00000154 0x3fca97b0 0x3fca6b64 0x3fc9a36c 0x3fc9a36c 0x3fca9b68 0x3fc9a364
3fca9b80: 0x00000018 0xbc2be427 0x91886245 0x3fca9b68 0x00000000 0x00000001 0x3fca8b64 0x6e69616d
3fca9ba0: 0x8c401b00 0xc9c2ac75 0x004187e7 0x00000000 0x3fca9b60 0x00000001 0x00000000 0x00000000
3fca9bc0: 0x00000000 0x00000000 0x3fca71d8 0x3fca7240 0x3fca72a8 0x00000000 0x00000000 0x00000001
3fca9be0: 0x00000000 0x00000000 0x00000000 0x4215af82 0x00000000 0x00000000 0x00000000 0x00000000
3fca9c00: 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000 0x00000000



ELF file SHA256: 3895d7f8f56ffd09
ELF file SHA256: 3895d7f8f56ffd09

Rebooting...
ESP-ROM:esp32c3-api1-20210207
Build:Feb  7 2021
rst:0x3 (RTC_SW_SYS_RST),boot:0xc (SPI_FAST_FLASH_BOOT)
Saved PC:0x403806ba
0x403806ba: esp_restart_noos_dig at D:/Frameworks/esp-idf-v4.4.2/components/esp_system/esp_system.c:53 (discriminator 1)
```

### More Information.

[sdkconfig.txt](https://github.com/espressif/esp-idf/files/10930771/sdkconfig.txt)

**Comment by hipal31** [comment]
@JiaLWang Can you help with this?

**Comment by hipal31** [comment]
@xiewenxiang Can you help me with this?

**Comment by xiewenxiang** [comment]
@hipal31 Please provide the elf file corresponding to the previous log.

**Comment by hipal31** [comment]
@xiewenxiang please find [elf file](https://github.com/espressif/esp-idf/files/10986767/BLE.elf.txt)

**Comment by hipal31** [comment]
@xiewenxiang have you gone through the elf file?

**Comment by hipal31** [comment]
@xiewenxiang any update on this?
Can anyone help?

**Comment by hipal31** [comment]
@xiewenxiang  any update?

**Comment by MingyaoLiu** [comment]
i think i have the same issue, try using a lower version of arduino-esp32. 
I have problem using platform io v6.1.0, revert to 5.3.0 no more panic.

**Comment by hipal31** [comment]
@igrr Can you help?

**Comment by esp-cjh** [comment]
Hi, @hipal31 
I can't convert the CPU register or backtrace value to code locations with your elf file.
You should find someone responsible for arduino-esp32 to help.

**Comment by hipal31** [comment]
@zhp0406 Can you please help me with this?

**Comment by zhp0406** [comment]
@hipal31 
I am currently reproducing the issue.I could not checkout to v4.4.4-1-g8a624b9b5f,
could you show the version of IDF you are using?

**Comment by hipal31** [comment]
@zhp0406 
You can checkout released version v4.4.4 for reproducing the issue.

**Comment by zhp0406** [comment]
@hipal31 
I could not reproduce the issue using esp-idf,
are you using arduino-ide? https://randomnerdtutorials.com/installing-the-esp32-board-in-arduino-ide-windows-instructions/

The following is my log, and I didn't find any issues.

I (30) boot: ESP-IDF v4.4.4 2nd stage bootloader
I (30) boot: compile time 11:11:25
I (30) boot: chip revision: v0.3
I (32) boot.esp32c3: SPI Speed      : 80MHz
I (37) boot.esp32c3: SPI Mode       : DIO
I (42) boot.esp32c3: SPI Flash Size : 2MB
I (46) boot: Enabling RNG early entropy source...
I (52) boot: Partition Table:
I (55) boot: ## Label            Usage          Type ST Offset   Length
I (63) boot:  0 nvs              WiFi data        01 02 00009000 00006000
I (70) boot:  1 phy_init         RF data          01 01 0000f000 00001000
I (78) boot:  2 factory          factory app      00 00 00010000 00100000
I (85) boot: End of partition table
I (89) esp_image: segment 0: paddr=00010020 vaddr=3c0c0020 size=2aec0h (175808) map
I (126) esp_image: segment 1: paddr=0003aee8 vaddr=3fc90400 size=02e98h ( 11928) load
I (128) esp_image: segment 2: paddr=0003dd88 vaddr=40380000 size=02290h (  8848) load
I (133) esp_image: segment 3: paddr=00040020 vaddr=42000020 size=b56c4h (743108) map
I (258) esp_image: segment 4: paddr=000f56ec vaddr=40382290 size=0dfa8h ( 57256) load
I (274) boot: Loaded app from partition at offset 0x10000
I (274) boot: Disabling RNG early entropy source...
I (286) cpu_start: Pro cpu up.
I (295) cpu_start: Pro cpu start user code
I (296) cpu_start: cpu freq: 160000000
I (296) cpu_start: Application information:
I (298) cpu_start: Project name:     onoff_server
I (304) cpu_start: App version:      v4.4.4
I (309) cpu_start: Compile time:     Sep 11 2023 11:11:22
I (315) cpu_start: ELF file SHA256:  ef7e316c7061326a...
I (321) cpu_start: ESP-IDF:          v4.4.4
I (326) heap_init: Initializing. RAM available for dynamic allocation:
I (333) heap_init: At 3FC9A370 len 000423A0 (264 KiB): DRAM
I (339) heap_init: At 3FCDC710 len 00002950 (10 KiB): STACK/DRAM
I (346) heap_init: At 50000020 len 00001FE0 (7 KiB): RTCRAM
I (352) spi_flash: detected chip: generic
I (357) spi_flash: flash io: dio
W (361) spi_flash: Detected size(4096k) larger than the size in the binary image header(2048k). Using the size in the binary image header.
I (374) sleep: Configure to isolate all GPIO pins in sleep state
I (380) sleep: Enable automatic switching of GPIO sleep configuration
I (388) coexist: coexist rom version 9387209
I (393) cpu_start: Starting scheduler.
I (397) EXAMPLE: Initializing...
I (397) gpio: GPIO[8]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0 
I (407) gpio: GPIO[8]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0 
I (417) gpio: GPIO[8]| InputEn: 0| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:0 
I (427) BTDM_INIT: BT controller compile version [02bfb7f]
I (427) phy_init: phy_version 950,11a46e9,Oct 21 2022,08:56:12
I (467) system_api: Base MAC address is not set
I (467) system_api: read default base MAC address from EFUSE
I (467) BTDM_INIT: Bluetooth MAC: 58:cf:79:ea:77:32

I (547) EXAMPLE: ESP_BLE_MESH_PROV_REGISTER_COMP_EVT, err_code 0
I (557) EXAMPLE: ESP_BLE_MESH_NODE_PROV_ENABLE_COMP_EVT, err_code 0
I (557) EXAMPLE: BLE Mesh Node initialized

**Comment by hipal31** [comment]
@zhp0406 Did you add arduino-esp32 as a component in your project.
https://espressif-docs.readthedocs-hosted.com/projects/arduino-esp32/en/latest/esp-idf_component.html like this?

**Comment by zhp0406** [comment]
@hipal31 
Did you make any modifications to the 'onoff_server example'?

**Comment by hipal31** [comment]
@zhp0406 NO

**Comment by hipal31** [comment]
Issue is Solved in newest version of Arduino-esp32(v2.0.13) and esp-idf(v4.4.4.)
